packages/frontend/src/features/metricsCatalog/components/MetricsTable.tsx

```diff
@@ -119,10 +120,18 @@ export const MetricsTable = () => {
         enableTopToolbar: true,
         mantinePaperProps: {
             shadow: undefined,
+            sx: {
+                border: `1px solid ${theme.colors.gray[2]}`,
+                borderRadius: theme.spacing.sm, // ! radius doesn't have rem(12) -> 0.75rem
```

Comment:
To be fixed by: [Fix conflicting styling in figma docs](https://github.com/lightdash/lightdash-internal-work/issues/2215)